server/lib/host-metrics.js

```diff
@@ -258,6 +258,30 @@ ORDER BY DATE_TRUNC(:timeUnit, t1."createdAt")`,
   return orderBy(mergedTimeSeries, 'date');
 }
 
+export async function getTotalMoneyManagedTimeSeries(host, { startDate, endDate, timeUnit } = {}) {
+  const results = await sequelize.query(
+    `SELECT SUM(t1."amountInHostCurrency") as "_amount", t1."hostCurrency" as "_currency", DATE_TRUNC(:timeUnit, t1."createdAt") as "date"
+FROM "Transactions" as t1
+WHERE t1."HostCollectiveId" = :HostCollectiveId
+AND t1."kind" != 'PLATFORM_FEE'
+AND t1."kind" != 'PLATFORM_TIP'
+AND t1."kind" != 'HOST_FEE'
```

Comment:
Why filter these? Does it match the results from https://github.com/opencollective/opencollective-api/blob/9588f9e3b746ec25796a5877810c1b1be82232ed/server/lib/budget.js#L137? 

Can we add a test for this, ideally comparing the results of `getTotalMoneyManagedAmount` and `getTotalMoneyManagedTimeSeries` to make sure they correspond?

Reply:
I've added a test and also did some minor adjustments; previously we had a mistake when calculating this stat since we where summing up both credits and debits. 🤯